tests/multimodel/test_hopf.py

```diff
@@ -146,24 +146,23 @@ def test_compare_w_neurolib_native_model(self):
         Compare with neurolib's native Hopf model.
         """
         # run this model - default is diffusive coupling
-        fhn_multi = HopfNetwork(self.SC, self.DELAYS, x_coupling="diffusive", seed=SEED)
-        multi_result = fhn_multi.run(DURATION, DT, ZeroInput(DURATION, DT).as_array(), backend="numba")
+        hopf_multi = HopfNetwork(self.SC, self.DELAYS, x_coupling="diffusive", seed=SEED)
+        multi_result = hopf_multi.run(DURATION, DT, ZeroInput(DURATION, DT).as_array(), backend="numba")
         # run neurolib's model
-        fhn_neurolib = HopfModel(Cmat=self.SC, Dmat=self.DELAYS, seed=SEED)
-        fhn_neurolib.params["duration"] = DURATION
-        fhn_neurolib.params["dt"] = DT
+        hopf_neurolib = HopfModel(Cmat=self.SC, Dmat=self.DELAYS, seed=SEED)
+        hopf_neurolib.params["duration"] = DURATION
+        hopf_neurolib.params["dt"] = DT
         # there is no "global coupling" parameter in MultiModel
-        fhn_neurolib.params["K_gl"] = 1.0
+        hopf_neurolib.params["K_gl"] = 1.0
         # delays <-> length matrix
-        fhn_neurolib.params["signalV"] = 1.0
-        fhn_neurolib.params["coupling"] = "diffusive"
-        fhn_neurolib.params["sigma_ou"] = 0.0
-        fhn_neurolib.params["xs_init"] = fhn_multi.initial_state[::2][:, np.newaxis]
-        fhn_neurolib.params["ys_init"] = fhn_multi.initial_state[1::2][:, np.newaxis]
-        fhn_neurolib.run()
+        hopf_neurolib.params["signalV"] = 1.0
+        hopf_neurolib.params["coupling"] = "diffusive"
+        hopf_neurolib.params["sigma_ou"] = 0.0
```

Comment:
I'm guessing `params["sigma_ou"] > 0.0`  is nontrivial...

Reply:
I haven't with noise so far... too nontrivial with all the seeds lying around, I guess you never have the same noise realisations without hacking deep into the framework...
I just did tests with noise in notebooks - e.g. compared ALN run with OU from `neurolib`'s native and from builder... I have some notebooks planned for multimodels - I wanted to do noise-comparison of at least ALN and maybe FHN.. are you ok with this being just in the notebook for visual inspection and not in the rigorous test?

Reply:
I'm ok with it, having the exact same noise vectors is probably tricky, yes.... it could be easier for fhn which has only one mass, e.g. one noise vector. in neurolib right now the noise vector is generated before random parameters afaik. which means that setting the seed in multimodel just before generating the noise vector (of same length) should yield the same noise.....